benchpress/jobs/jobs.yml

```diff
@@ -72,3 +72,23 @@
     scale-factor: 1
     runtime: 1
 
+- benchmark: gapbs_bc
```

Comment:
This benchmark uses OpenMP. Do you think we should let it default OMP threads or we should specify that as one of the benchmark inputs?

Reply:
Today, the user can set the `OMP_NUM_THREADS` variable in their shell, and the benchmark should pick it up. But to be more explicit, we could potentially add support for environment variables, perhaps something like:

```yaml
benchmark: gapbs_bc
  name: gapbs_bc
  description: GAP bc benchmark
  env:
    OMP_NUM_THREADS: 4
  args:
    - -u 21
    - -n 1
```